megamek/src/megamek/client/ui/swing/tooltip/UnitToolTip.java

```diff
@@ -2103,6 +2226,14 @@ private static StringBuilder getMovement(Entity entity) {
                 String attr = String.format("FACE=Dialog COLOR=%s", UIUtil.toColorHexString((GUIP.getWarningColor())));
                 sMove += UIUtil.tag("FONT", attr, sArmor);
             }
+            // Display SI for Aerodynes
+            if (entity instanceof IAero) {

```

Comment:
`public class LandAirMek extends BipedMek implements IAero, IBomber`

IAero is the interface for all Aero units, LAMs also implement IAero and they do not have SI.